src/puppet-dom.js

```diff
@@ -50,7 +35,7 @@
       this.element.removeEventListener('puppet-redirect-pushstate', this.historyHandler);
     };
 
-    options.remoteUrl = options.remoteUrl || getRemoteUrlFromCookie() || window.location.href;
+    options.remoteUrl = options.remoteUrl || window.location.href;
```

Comment:
Nay... Let's keep it as is, because that would be a big refactor. Out of scope in this issue.